tests/dashboard/test_permissions.py

```diff
@@ -0,0 +1,247 @@
+from __future__ import unicode_literals
+
+from django.core.urlresolvers import reverse
+from django.contrib.auth.models import Group
+
+from saleor.userprofile.models import User
+from saleor.dashboard.group.forms import GroupPermissionsForm
+from saleor.dashboard.staff.forms import UserGroupForm
+
+
+def test_superuser_permissions(admin_user):
+    assert admin_user.has_perm("product.view_product")
+    assert admin_user.has_perm("product.edit_product")
+
+
+def test_superuser(admin_user):
+    assert isinstance(admin_user, User)
+
+
+def test_staff_list_view(admin_client):
+    response = admin_client.get('/dashboard/staff/')
```

Comment:
Again, you should always use `reverse` for getting URLs.

Reply:
Yes, first test. I was checking if this works as i expect and forget about it. Deleted